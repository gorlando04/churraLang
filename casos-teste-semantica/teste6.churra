amigo "Felipe" contribui 100 vegetariano
amigo "Jo√£o" contribui 100 vegetariano
amigo "Mariana" contribui 100 

item carne "picanha" preco_kg 10 quantidade_kg 1
item carne "fraldinha" preco_kg 20 quantidade_kg 2
item carne "picanha" preco_kg 30 quantidade_kg 3
item bebida preco_L 10 quantidade_L 6
item comida preco_unit 25 quantidade 2

rateio igual