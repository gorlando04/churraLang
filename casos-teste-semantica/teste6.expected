❌ Erro: [SEMÂNTICO] A carne 'picanha' já foi declarada anteriormente.
